ooiui/static/js/views/asset_management/EventViewPage.js

```diff
@@ -187,24 +169,6 @@ var EventViewPage = Backbone.View.extend({
 
     initializeEvent:function(){
         var self = this;
-        //datepickers with min/max validation
-        $( "#startdate_d" ).datepicker({
-          changeMonth: true,
-          numberOfMonths: 2,
-          maxDate: "+1m +1w",
-          onClose: function( selectedDate ) {
-            $( "#enddate_d" ).datepicker( "option", "minDate", selectedDate );
-          }
-        });
-        $( "#enddate_d" ).datepicker({
-          changeMonth: true,
-          numberOfMonths: 2,
-          maxDate: "+1m +1w",
-          onClose: function( selectedDate ) {
-            $( "#startdate_d" ).datepicker( "option", "maxDate", selectedDate );
-          }
-        });
-
```

Comment:
@DanielJMaher , @ssontag55 This was causing the date to re render with the current date +1m and +1w, and not the event start date.  SO I removed the date picker in lieu of the correct start/end dates.  Adding the date picker back in would be nice, but for this demo I think it's best not to have conflicting dates.